realm_reference_docs_help.md

```diff
@@ -0,0 +1,36 @@
+# Realm Docs Team SDK Reference Manual (API Docs) Collaboration
+
+# Motivation
+
+- API documentation could use some love/attention
+- Docs folks can add an extra set of eyes to make sure that descriptions are current, language is clear, etc.
+- Wordings, sentence structure, descriptions
+
+# Product Description
+
+## Proposed Solution
+
+- docs writers help with API docs, updating them when they notice gaps or inconsistencies
+- tickets to improve API docs independently
+- docs writers as a highly suggested LGTM on SDK work (anywhere that touches code or logic that impacts API docs)
+    - help with wording suggestions
+    - help by asking questions, suggesting additional information to add
+
+## Goals
+
+- build comradery and relationships between docs and the realm SDK teams
+- improve quality of API docs
+- improve wording of API docs, make more consistent with docs docs
+
+## Non-Goals
+
+- docs team will not block SDK work -- we're just helping out, trying to make the API docs even better
+
+## Use Cases
+
+- all realm-<language> sdk repos
```

Comment:
I know this is an issue with driver teams because of how they release their API docs alongside the driver version in Github and never republish older versions. Does Realm have the same issue?

Reply:
Honestly I'm not really sure how often older reference manuals get updated now. I suspect that there are common issues with the driver teams (maybe both groups could implement the same solution?)